src/puppetlabs/ezbake/core.clj

```diff
@@ -819,6 +819,14 @@ Additional uberjar dependencies:
   (action "stage" lein-project build-target)
   (exec/exec "rake" "package:bootstrap" :dir staging-dir)
   (let [downstream-job nil
+        rake-call ["rake" "pl:jenkins:trigger_build_local_auth"]]
```

Comment:
So if I call `lein with-profile ezbake ezbake build`, that will invoke this new way of building? How do the pipelines kick off the ezbake build? I'm wondering if this behavior change should trigger a new major version? If the existing ci pipelines build with a different call that they can opt into beyond just bumping the ezbake version, that'd be fantastic.

Also, we should update the docs. I know I refer to https://confluence.puppetlabs.com/display/ENG/How+To+build+Puppet+Server+from+a+local+topic+branch a lot when building locally with ezbake changes

Reply:
None of the pipelines use `lein with-profile ezbake ezbake build` but rather call `uber_ship` directly, so this would only affect testing locally, and the change should be completely transparent with the exception of needing to set the `JENKINS_USER_AUTH` environment variable to auth with jenkins. I tried to make this as straightforward a transition as possible.

Reply:
The only change for running from an ezbake topic branch is setting `JENKINS_USER_AUTH`. Once it's merged/released I'll update confluence. It will also fail very informatively and let you know that this var is needed and how to find/set it.